src/styles/colors.scss

```diff
@@ -0,0 +1,9 @@
+:root {
+  // Default Colors
+  --white: #ffffff;
+  --black: #000000;
+  --danger: red;
```

Comment:
Lets use Hex colors only red differs browsers to browser

Reply:
No problem, --danger var's been updated to hex code.